packages/neuron-ui/src/containers/Notification/index.tsx

```diff
@@ -79,7 +80,8 @@ export const NoticeContent = () => {
   const dispatch = useDispatch()
   const [t, i18n] = useTranslation()
   useOnLocaleChange(i18n)
-  useGlobalNotifications(dispatch)
+  const [hasDismiss, setHasDismiss] = useState(false)
+  useGlobalNotifications(dispatch, hasDismiss)
```

Comment:
`hasDismiss` can be used to hide the `notification component` so the logic of `onBackUp` can be intact.

Reply:
I didn't get the point about `onBackUp can be intact`

Reply:
My fault, `useGlobalNotifications` can be intact because `dismiss` will not be triggered second time if the dialog is hidden by `hasDismiss`